linter/README.md

```diff
@@ -0,0 +1,143 @@
+# Linter
+
+The following task is used to provide linter support for various languages:-
+
+- `checkmake` (Makefile lint)
+- `yamllint` (YAML linter)
+- `mdlint` (Markdown linter)
+- `shellcheck` (Shell script linter)
+- `rubocop` (Ruby linter)
+- `tslint` (TypeScript linter)
+
+## Installing the Tasks
+
+### Installing the `check-make` Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/linter/check-make.yaml
```

Comment:
Once the task is moved to the task directory can you please change the install path command  as well